release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -3326,6 +3326,62 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+[id="ocp-4-14-34_{context}"]
+=== RHSA-2024:4960 - {product-title} 4.14.34 bug fix and security update
+
+Issued: 08 August 2024
+
+{product-title} release 4.14.34, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:4960[RHSA-2024:4960] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:4963[RHSA-2024:4963] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.14.34 --pullspecs
+----
+
+[id="ocp-4-14-34-enhancements_{context}"]
+==== Enhancements
+
+[id="ocp-4-14-34-timeout-ingress_{context}"]
+===== Adding connectTimeout tuning option to Ingress Controller API
+
+* The IngressController API is updated with a new tuning option, `ingresscontroller.spec.tuningOptions.connectTimeout`, which defines how long the router will wait for a response when establishing a connection to a backend server. (link:https://issues.redhat.com/browse/OCPBUGS-36555[*OCPBUGS-36555*])
+
+[id="ocp-4-14-34-insights-operator-collecting-prometheus-alertmanager_{context}"]
+===== Collecting Prometheus and AlertManager resources with the Insights Operator
+
+* The Insights Operator (IO) now collects the `Prometheus` and `AlertManager` resources in addition to the `openshift-monitoring` custom resource. (link:https://issues.redhat.com/browse/OCPBUGS-36380[*OCPBUGS-36380*])
```

Comment:
I don't think I've ever seen this operator abbreviated as "IO", so I'd recommend removing that

I also don't think that it's right to say that "openshift-monitoring" is a custom resource. Custom resources are usually styled in pascal case, like "AlertManager".

"openshift-monitoring" looks like it's probably a namespace. After looking at the bug, perhaps this would be more accurate?

```suggestion
* The Insights Operator now collects the `Prometheus` and `AlertManager` resources outside of the `openshift-monitoring` namespace. (link:https://issues.redhat.com/browse/OCPBUGS-36380[*OCPBUGS-36380*])
```